apps/documentation/src/routes/documentation/tutorial/components.mdx

```diff
@@ -63,8 +63,8 @@
 // ...
 ```
 
-Now the links work. Clicking on any of them we get redirected to the 404 page because we haven’t yet implemented the `pokemons/[pokemon]` page.
-As previously said CSS modules are enabled out of the box so let’s make those links a little bit nicer.
+Now the links work. Clicking on any of them, we get redirected to the 404 page because we haven’t yet implemented the `pokemons/[pokemon]` page.
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The name of this game or TV show is spelled with an accent. (POKEMON[1])
Suggestions: `Pokémon`
URL: https://en.wikipedia.org/wiki/Pok%C3%A9mon 
Rule: https://community.languagetool.org/rule/show/POKEMON?lang=en-US&subId=1
Category: GRAMMAR
<!-- __reviewdog__:Cg80Yjc0NmVmMmM4MTdjYzUSDExhbmd1YWdlVG9vbA== -->